### PR TITLE
sdl(all): Create SBOM for net packages 

### PR DESCRIPTION
**Pull request by jamdavi**
We need to create an SBOM to allow customers to see the dependencies of our software. This adds the task to do so.

### CODE REVIEW

File: vsts/build-release-artifacts.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # update .csproj file versions and create a release branch/commit
 name: BumpVersion_$(BuildID)_$(Date:yyyyMMdd)$(Rev:.r)
+pool:
```
Comment (jamdavi) [inline vsts/build-release-artifacts.yml:3]: The changes that are outside of the added task were validation errors exposed by ADO when testing this new build.

**Review by andyk-ms — APPROVED** [review]

**Pull request merged**